core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/UniqueIDJDBCUserStoreManager.java

```diff
@@ -3472,6 +3483,300 @@ protected UniqueIDPaginatedSearchResult doGetUserListWithID(Condition condition,
         return result;
     }
 
+    @Override
+    public String doGetGroupIdFromGroupName(String groupName) throws UserStoreException {
+
+        String sqlStmt = realmConfig.getUserStoreProperty(JDBCRealmConstants.GET_GROUP_ID_FROM_GROUP_NAME);
+
+        try (Connection dbConnection = getDBConnection();
+             PreparedStatement prepStmt = dbConnection.prepareStatement(sqlStmt)) {
+
```

Comment:
Shall we get rid not unwanted newlines to maintain consistency in the code?

Reply:
Addressed with https://github.com/wso2/carbon-kernel/pull/3847/commits/344924abcc35406833451b3a6c68ab68ec225912